config/templates/stack-template.json

```diff
@@ -1116,5 +1116,25 @@
     {{end}}
     {{end}}
 
+  },
+
+  "Outputs": {
+    "WorkerSecurityGroup" : {
+      "Description" : "The security group assigned to worker nodes",
+      "Value" :  { "Ref" : "SecurityGroupWorker" },
+      "Export" : { "Name" : {"Fn::Sub": "${AWS::StackName}-WorkerSecurityGroup" }}
+    },
```

Comment:
@camilb Good catch! Thanks a lot :bow:

I believe I've made this kind of mistakes several times so far.
To reduce the chance of doing the same again, firstly I've [prepared tests to cover these cases](https://github.com/coreos/kube-aws/pull/106/commits/d68b0350f8c444d42254d745c837afabd92a4a1e) and then [fixed the issue](https://github.com/coreos/kube-aws/pull/106/commits/940ba3f22161eea282d1652694b8a21ef506c439).

I wish I could cover more combinations of settings, options in our e2e test but it had been a bit costly in time and AWS charge 😈
I'd like to just keep covering more cases in unit testing like I did this time for now.